### PR TITLE
feat: add Osmosis chain / warp definitions

### PR DESCRIPTION
**Pull request by byeongsu-hong**

### CODE REVIEW

File: src/consts/tokens.ts [hunk 1]
```diff
@@ -5,6 +5,60 @@ import { Chains, TokenConnectionType, TokenStandard, WarpCoreConfig } from '@hyp
...
+      symbol: 'milkTIA',
+      name: 'Hyperlane Bridged milkTIA',
+      addressOrDenom: '0x32474653127048d9fC20000e21dEd396b47968E8',
+      connections: [
+        {
+          token: 'cosmos|osmosis|osmo17xuecsykqw2xcxwv8cau7uy4hgdwqt0u4qxflyc2yshhggpazfjs6kfqd3',
+        },
+      ],
+    },
+
```
Comment (jmrossy) [inline src/consts/tokens.ts:61]: You're missing the `//#endregion OSMOSIS` closing comment here

**Review by jmrossy — COMMENTED** [review]
Look good! Just one small thing

File: src/consts/tokens.ts (hunk 1 above)
Comment (byeongsu-hong) [reply src/consts/tokens.ts:61]: Oh! fixing it now :)

Comment (byeongsu-hong) [reply src/consts/tokens.ts:61]: fixed :)

**Comment by jmrossy** [comment]
Thank you @byeongsu-hong. I'm merging this to a feature branch so we can prepare a test deployment. We'll merge into nexus when the time is right.

**Pull request merged**